### ISSUE TITLE
OriginateAction Timeout

### ISSUE THREAD

**Issue by Deantwo** [AsterNET/AsterNET#61]
See: [OriginateAction.cs#L141](https://github.com/skrusty/AsterNET/blob/f179276c916f1e10a4fdcdc851637da8556871ab/Asterisk.2013/Asterisk.NET/Manager/Action/OriginateAction.cs#L141)

The summery starts by saying that the value is in seconds, then it goes on to say that the default value is "30000 meaning 30 seconds" which seems weird.
But when omitted it seems to be set to 0, since the action throws timeout right after being called.

Also when the value is set to 3'000'000 it does not last for 34.72 days.

**Code sample:**
```
ManagerResponse originateResponse = myManagerConn.SendAction(new OriginateAction()
{
    Channel = "SIP/" + sip,
    Exten = exten,
    Context = context,
    Priority = "1",
    //Timeout = 3000000,
    CallerId = callerId + " <" + sip + ">"
});
```

**TL;DR checklist:**
- [ ] Summery says seconds while that is clearly not the case.
- [ ] When left undefined the value seems to be 0 rather than the mentioned default.

**Comment by skrusty** [comment]
It's Milliseconds, the summary is wrong.

The default may be referencing Asterisk's default, where was when omitted from AsterNET, it's replaced by 0. I would need to check that however.

**Comment by Deantwo** [comment]
> It's Milliseconds, the summary is wrong.

That does indeed make more sense.
But I guess there is some kind of limit, sense my phone didn't ring for 50 minutes.

**Comment by Deantwo** [comment]
Are you about to commit a fix to this? Or why are you closing it?

**Comment by skrusty** [comment]
Unless you wish to submit a PR to correct the issue, we intend to fix it as part of version 2 which will auto generate documentation based off the Asterisk XML docs. Not perfect, but there are so many inconstancies in the current documentation that it's 
1) hard for us to maintain 
2) work that could be better focused elsewhere.

Again however, feel free to push over amendments!

**Comment by Deantwo** [comment]
As long as you fix the default timeout value, in "version 2" and/or here.
The documentation is a easy fix, but it doesn't matter too much.

**Comment by skrusty** [comment]
So you don't fancy contributing that back to our open source community?

**Comment by Deantwo** [comment]
I would, but I haven't had the time to dig into the code for it.
You said you were gonna look into it, so I haven't felt the need either. ^^;

If I have the time and figure it out, I will totally toss a PR your way.
